core/src/test/scala/org/apache/spark/scheduler/BlacklistTrackerSuite.scala

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.scheduler
+
+import org.apache.spark.{SparkConf, SparkFunSuite}
+import org.apache.spark.internal.config
+
+class BlacklistTrackerSuite extends SparkFunSuite {
+
+  test("blacklist still respects legacy configs") {
+    val legacyKey = config.BLACKLIST_LEGACY_TIMEOUT_CONF.key
+
+    {
+      val localConf = new SparkConf().setMaster("local")
+      assert(!BlacklistTracker.isBlacklistEnabled(localConf))
+      localConf.set(legacyKey, "5000")
+      assert(BlacklistTracker.isBlacklistEnabled(localConf))
+      assert(5000 === BlacklistTracker.getBlacklistTimeout(localConf))
+
+      localConf.set(legacyKey, "0")
+      assert(!BlacklistTracker.isBlacklistEnabled(localConf))
+    }
+
+    {
```

Comment:
good point, but I'll also turn blacklisting off by default so this paritcular test will just reorganized a little.